dcm4che-tool/dcm4che-tool-stowrs/src/main/java/org/dcm4che3/tool/stowrs/StowRS.java

```diff
@@ -163,6 +164,7 @@ private static Attributes configureKeys(CommandLine cl) {
 
     @SuppressWarnings("unchecked")
     public static void main(String[] args) {
+        BasicConfigurator.configure();
```

Comment:
Please do not call BasicConfigurator.configure() directly. Instead the configuration should be done using the `log4j.properties` file located in `dcm4che-assembly/src/etc/stowrs/log4j.properties`.
I'm not sure why that doesn't work already correctly. 
@vrindanayak Could you have a look why the stowrs tool behaves differently than the other tools?